mjpc/tasks/humanoid/interact/contact_keyframe.h

```diff
@@ -0,0 +1,56 @@
+#ifndef CONTACT_KEYFRAME_H
+#define CONTACT_KEYFRAME_H
```

Comment:
Please use the standard header definition style and naming convention that we use throughout the project. See example [here](https://github.com/google-deepmind/mujoco_mpc/blob/c82f6a6bd45dffdd02cfd073f126dc2edcb16442/mjpc/tasks/humanoid/walk/walk.h#L15).